src/network/core_utils.rs

```diff
@@ -832,7 +832,7 @@ impl CoreUtils {
         if let Err(err) = handle
             .link()
             .add()
-            .veth(host_veth.to_string(), container_veth.to_string())
+            .veth(container_veth.to_string(), host_veth.to_string())
```

Comment:
@baude Sorry i think i merged it. Just one doubt I understand that swapping `veth` order is fixing the issue but not sure how ? Is it a `kernel` bug ?

Reply:
I am not sure tbh. I just tried until it works. I have a script with only `ip` commands to reproduce the bridge setup. However with `ip link add` the order did not matter so it is not only the kernel. I think the `ip link` command adds some netlink attributes and our rust code does not.